### PR TITLE
SFA-2: Create Student-Questions-Model

### PR DESCRIPTION
**Pull request by xuantruongcox**
### Please read and mark the following check list before creating a pull request (check one with "x"):

 - [ ] I read and followed the [CONTRIBUTING.md](https://github.com/akveo/ngx-admin/blob/master/CONTRIBUTING.md) guide.
 - [ ] I read and followed the [New Feature Checklist](https://github.com/akveo/ngx-admin/blob/master/DEV_DOCS.md#new-feature-checklist) guide.
 
 #### Short description of what this resolves:

### CODE REVIEW

File: src/app/@core/models/question-controls/question-dropdown.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import { QuestionBase } from '../question-student';
+
+export class DropdownQuestion extends QuestionBase<string>{
```
Comment (ProjectHocVienCongGiao) [inline src/app/@core/models/question-controls/question-dropdown.ts:3]: This should be renamed to BaseQuestion

File: src/app/guest-pages/guest-pages.component.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+        <nb-layout>
+            <nb-layout-column class="bg-white">
+
+            </nb-layout-column>
+        <nb-layout>
+    `,
+    styleUrls: ['./guest-pages.component.scss'],
+})
+
+export class GuestPagesComponent {
```
Comment (ProjectHocVienCongGiao) [inline src/app/guest-pages/guest-pages.component.ts:15]: what does this component do?

**Review by ProjectHocVienCongGiao — CHANGES_REQUESTED** [review]
The test is failing

File: src/app/guest-pages/guest-pages.component.ts (hunk 2 above)
Comment (xuantruongcox) [reply src/app/guest-pages/guest-pages.component.ts:15]: sorry sir, this's layout for guest

File: src/app/@core/mock/question-control.service.ts [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import { Injectable } from '@angular/core';
+import { FormControl, FormGroup, Validators } from '@angular/forms';
+
+import { BaseQuestion } from '../models/question-student';
```
Comment (ProjectHocVienCongGiao) [inline src/app/@core/mock/question-control.service.ts:4]:
```suggestion
import { BaseQuestion } from '../models/student-question';
```

**Review by ProjectHocVienCongGiao — COMMENTED** [review]
Some grammar mistake

**Review by nguyenhongphat0 — APPROVED** [review]

**Review by nguyenhongphat0 — APPROVED** [review]

**Review by nguyenhongphat0 — APPROVED** [review]

**Pull request merged**